### ISSUE TITLE
Imagine verification

### ISSUE THREAD

**Issue by NiladriXX** [NiladriXX/QTECHSQUBITS#1]
# Import the necessary libraries

import tensorflow as tf
import numpy as np
input_size = 784  # Assuming images are 28x28 pixels and flattened to 784-dimensional vectors
hidden_size = 256
output_size = 10

# Define the neural network architecture
def create_model():
    model = tf.keras.Sequential([
        tf.keras.layers.Dense(hidden_size, activation='relu', input_shape=(input_size,)),
        tf.keras.layers.Dense(hidden_size, activation='relu'),
        tf.keras.layers.Dense(output_size, activation='softmax')
    ])
    return model

model = create_model()
num_images = 1000

# Generate random labels for the images (0 to 9)
labels = np.random.randint(0, output_size, num_images)

# Convert labels to one-hot encoding
verification_matrix = tf.one_hot(labels, output_size)

# To visualize the neural network architecture

import matplotlib.pyplot as plt
import tensorflow as tf

def plot_model(model):
    tf.keras.utils.plot_model(model, show_shapes=True, show_layer_names=True)
    plt.show()

model = create_model()
plot_model(model)

# To generate random images:

import numpy as np
import matplotlib.pyplot as plt

def generate_random_image():
    image_size = 28
    random_image = np.random.rand(image_size, image_size)
    return random_image

num_images_to_generate = 5
for i in range(num_images_to_generate):
    random_image = generate_random_image()
    plt.imshow(random_image, cmap='gray')
    plt.axis('off')
    plt.show()

# Happy coding! Credit: Niladri
